include/lbann/layers/regularizers/dropout.hpp

```diff
@@ -54,10 +54,7 @@ class dropout : public regularizer_layer {
   #if defined(LBANN_HAS_CUDNN) && !defined(LBANN_SEQUENTIAL_CONSISTENCY)
```

Comment:
The guard for && !defined(LBANN_SEQUENTIAL_CONSISTENCY) should probably go away, or we need to prevent dropout from being marked as a not legal for GPUs when SEQUENTIAL consistency is requested.  Otherwise this causes a runtime error.